fedimint-server/src/consensus/server.rs

```diff
@@ -328,12 +328,12 @@ impl ConsensusServer {
     pub async fn run_session(&self, session_index: u64) -> anyhow::Result<()> {
         // if all nodes are correct the session will take 45 to 60 seconds. The
         // more nodes go offline the longer the session will take to complete.
-        const EXPECTED_ROUNDS_PER_SESSION: usize = 45 * 4;
+        const EXPECTED_ROUNDS_PER_SESSION: usize = 45 * 20;
         // this constant needs to be 3000 or less to guarantee that the session
         // can never reach MAX_ROUNDs.
         const EXPONENTIAL_SLOWDOWN_OFFSET: usize = 3 * EXPECTED_ROUNDS_PER_SESSION;
-        const MAX_ROUND: u16 = 5000;
-        const ROUND_DELAY: f64 = 250.0;
+        const MAX_ROUND: u16 = 5_000;
+        const ROUND_DELAY: f64 = 50.0;
```

Comment:
I wonder if we could calculate all the values from a given round delay and a target length of one session. That would make changing these values easier.

Reply:
The thing is that MAX_ROUND, ROUND_DELAY, EXPECTED_ROUNDS the number of peers and the maximum size of a unit all play into the maximal throughput of kb of txs per second, maximal ram consumption per session, session time bounds and latency. Besides ROUND_DELAY we need consensus on all constants so they can not be adapted after the federation is running. So tuning this is quite tricky here and can not be changed afterwards, therefore I opted to set a sensible default instead of exposing this complexity to the guardians. I think it will probably make sense to make this configurable in the future, especially when can simplify this interplay by upstreaming a small change to Aleph BFT.